src/integrationTest/kotlin/net/corda/test/DriverBasedTest.kt

```diff
@@ -0,0 +1,26 @@
+package net.corda.test
+
+import net.corda.core.getOrThrow
+import net.corda.core.node.services.ServiceInfo
+import net.corda.node.services.transactions.SimpleNotaryService
+import net.corda.testing.DUMMY_BANK_A
+import net.corda.testing.DUMMY_BANK_B
+import net.corda.testing.DUMMY_NOTARY
+import net.corda.testing.driver.driver
+import org.junit.Test
+
+class DriverBasedTest {
+    private companion object {
+        val nodeALegalName = DUMMY_BANK_A.name
+        val nodeBLegalName = DUMMY_BANK_B.name
+    }
+
+    @Test
+    fun `run driver test`() {
+        driver(isDebug = true) {
+            startNode(DUMMY_NOTARY.name, setOf(ServiceInfo(SimpleNotaryService.type))).getOrThrow()
+            startNode(nodeALegalName).getOrThrow()
+            startNode(nodeBLegalName).getOrThrow()
+        }
```

Comment:
Maybe go a step further here and show how to grab the RPC connection to actually interact with the node?

Reply:
Yeah, to be a test it needs an assert. I'll add something in to assert on.

Reply:
Actually, this applies to the other test types (contract and flow). We don't want to write any actual tests, since this is a template, not an example. But getting the RPC connection seems like a bit of boilerplate they'd need anyway.